File: cursoPOOUber/PHP/card.php
<?php
require_once('./payment.php');

class Card extends Payment
{
    public $number;
    public $cvv;
    public $date;

    public function __contruct($id, $number, $cvv, $date)
    {
        parent::__construct($id);
        $this->$number = $number;
        $this->$cvv = $cvv;
        $this->$date = $date;
    }
}

?>

File: cursoPOOUber/Java/Card.java
package Java;

class Card extends Payment {
    Integer number;
    Integer cvv;
    String  date;
    
    public Card(Integer id, Integer number, Integer cvv, String date) {
        super(id);
        this.number = number;
        this.cvv = cvv;
        this.date = date;
    }
    
}
